Xamarin.Forms.Core/ITextElement.cs

```diff
@@ -1,8 +1,10 @@
 using System;
+using System.ComponentModel;
 
 namespace Xamarin.Forms
 {
-	interface ITextElement
+	[EditorBrowsable(EditorBrowsableState.Never)]
+	public interface ITextElement
```

Comment:
you shouldn't be using this properties from renderers, but using SetValue. no need to be public